implementations/rust/ockam/ockam_command/src/influxdb/inlet/create.rs

```diff
@@ -1,25 +1,148 @@
 use crate::node::util::initialize_default_node;
-use crate::tcp::inlet::create::CreateCommand as InletCreateCommand;
+use crate::shared_args::OptionalTimeoutArg;
+use crate::tcp::inlet::create::{tcp_inlet_default_from_addr, tcp_inlet_default_to_addr};
+use crate::tcp::util::alias_parser;
+use crate::util::parsers::duration_parser;
+use crate::util::parsers::hostname_parser;
+use crate::util::{port_is_free_guard, print_warning_for_deprecated_flag_replaced};
 use crate::{Command, CommandGlobalOpts};
 use async_trait::async_trait;
+use clap::builder::FalseyValueParser;
 use clap::Args;
 use colorful::Colorful;
-use miette::miette;
+use miette::{miette, IntoDiagnostic};
+use ockam::identity::Identifier;
+use ockam::transport::SchemeHostnamePort;
 use ockam::Context;
+use ockam_abac::PolicyExpression;
+use ockam_api::address::extract_address_value;
+use ockam_api::cli_state::random_name;
 use ockam_api::colors::color_primary;
 use ockam_api::influxdb::{InfluxDBPortals, LeaseUsage};
 use ockam_api::nodes::models::portal::InletStatus;
 use ockam_api::nodes::BackgroundNodeClient;
-use ockam_api::{fmt_info, fmt_log, fmt_ok, fmt_warn, ConnectionStatus};
+use ockam_api::{fmt_info, fmt_log, fmt_ok, fmt_warn, CliState, ConnectionStatus};
 use ockam_core::api::{Reply, Status};
-use ockam_multiaddr::MultiAddr;
+use ockam_multiaddr::{proto, MultiAddr, Protocol};
+use ockam_node::compat::asynchronous::resolve_peer;
+use std::str::FromStr;
+use std::time::Duration;
 use tracing::trace;
 
 /// Create InfluxDB Inlets
 #[derive(Clone, Debug, Args)]
-pub struct InfluxDBCreateCommand {
+pub struct CreateCommand {
+    /// Assign a name to this InfluxDB Inlet
+    #[arg(id = "NAME", value_parser = alias_parser)]
+    pub name: Option<String>,
+
+    /// Node on which to start the InfluxDB Inlet.
+    #[arg(long, display_order = 900, id = "NODE_NAME", value_parser = extract_address_value)]
+    pub at: Option<String>,
+
+    /// Address on which to accept InfluxDB connections, in the format <scheme>://<address>:<port>.
```

Comment:
I think that we should keep the name `Address` for the full `<scheme>://<hostname>:<port>` format and then refer to each component separately with those names. For example "the default hostname is 127.0.0.1". 

Or maybe even say `base URL` instead of `Address`? (it's not really a URL because there's no resource path). This would leave the name `Address` for our multi-addresses like `/project/default`.

Reply:
I've updated args docs to use the same words consistently, not sure if that's all you meant here.